src/reducers/inputs.js

```diff
@@ -156,17 +156,10 @@ export const clearResourceURIMessage = (state) => {
 
 export const removeMyItem = (state, action) => {
   const newState = { ...state }
-  const reduxPath = action.payload.reduxPath
-  let level = 0
+  const reduxPath = action.payload
 
-  reduxPath.reduce((obj, key) => {
-    level++
-    if (level === reduxPath.length) {
-      delete obj[key].items[action.payload.id]
-    }
-
-    return obj[key]
-  }, newState)
+  const node = findNode(newState, reduxPath.slice(0, -1))
```

Comment:
I prefer the original way of doing this. You know, because the original way was more confusing and complicated.